frontend/public/src/lib/util.js

```diff
@@ -231,10 +231,9 @@ export const formatLocalePrice = (amount: number | null) => {
   return amount.toLocaleString("en-US", { style: "currency", currency: "USD" })
 }
 
-export const getFlexiblePriceForProduct = product => {
+export const getFlexiblePriceForProduct = (product:any) => {
```

Comment:
The type for this is in `flow/cartTypes` (it'll need to be updated slightly to accommodate the new field) but once that is changed this should be good to go.

Reply:
Let me know if my most recent change looks good